docs/security/unauthorized-sign-in.mdx

```diff
@@ -7,9 +7,7 @@ Clerk detects sign-in attempts from unrecognized devices to protect users from u
 
 ## Email notification for unauthorized access
 
-When a sign-in attempt is made from an unfamiliar device, Clerk notifies the account owner by email with details about the newly created session. The account owner can immediately revoke the session if it's recognized as unauthorized.
-
-The email notification users receive for unauthorized sign-ins varies depending on the instance's configuration and the application's billing plan.
+When a sign-in attempt is made from an unfamiliar device, Clerk notifies the account owner by email with details about the newly created session. The email notification varies depending on the instance's configuration and the application's billing plan.
```

Comment:
I've removed "The account owner can immediately revoke the session if it's recognized as unauthorized." as this is only available for certain supported instances. So now, we can combine the first and third sentence, and it flows very nicely.